### PR TITLE
feat(interact): update the feature of suite1.rs

### PR DESCRIPTION
**Pull request by happyhappy-jun**
previously function upload_modify_and_query have to deploy, interact with smart contract. Now, changed to simply interact with pre-deployed contract.

- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix
- [ ] Dependency, Environment, Build related update 
- [x] Documentation
- [ ] Other tiny fix 

## Describe your changes
- `suite1.rs` 의 컨트랙트를 배포하고 상호작용하는 기능이 단순 상호작용으로 변경되었습니다. 

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Leave reference about this pull request, if exist (ex. discord message, google docs, etc)
- [x] If it is a core feature, I have added thorough tests.

## Checklist after creating a pull request
- [x] Mention reviews in discord channel directly

### CODE REVIEW

File: interact/tests/suite1.rs [hunk 1]
```diff
@@ -47,6 +47,6 @@ async fn check_account() {
 #[tokio::test]
 async fn upload_modify_and_query() {
```
Comment (junha1) [inline interact/tests/suite1.rs]: 여기도 upload를 뺍시다.

**Review by junha1 — COMMENTED** [review]
커밋 제목이 `update the feature of suite1.rs` 가 아니라 `Skip contract upload step requirement of test` 등 구체적인게 낫지 않을까요?

**Comment by junha1** [comment]
그리고 `main` 리베이스가 안되어있는 듯 하네요.

**Comment by happyhappy-jun** [comment]
~~commit title 에 적으려다가 commit description 으로 뺐는데, 제목에 적는게 좋을까요?~~

https://github.com/postech-dao/near/pull/16#pullrequestreview-1033539413 -> 제목에도 적어주는게 좋을꺼 같네요 수정하겠습니다

https://github.com/postech-dao/near/pull/16#issuecomment-1179550763 ->  main rebase 완료

File: interact/tests/suite1.rs (hunk 1 above)
Comment (happyhappy-jun) [reply interact/tests/suite1.rs]: fixed in [521e1d8](https://github.com/postech-dao/near/pull/16/commits/521e1d80c8dd95d1b60382813777696b1b10fce5)

Comment (happyhappy-jun) [reply interact/tests/suite1.rs]: fixed [521e1d8](https://github.com/postech-dao/near/pull/16/commits/521e1d80c8dd95d1b60382813777696b1b10fce5)

Comment (happyhappy-jun) [reply interact/tests/suite1.rs]: fixed [332031b](https://github.com/postech-dao/near/pull/16/commits/332031b45c69b57a281e42225dd27b7684835905)

**Review by junha1 — APPROVED** [review]

**Pull request merged**